### DISCUSSION TITLE
VNC server on clients

### DISCUSSION THREAD

**Discussion by mecrider** [ltsp/ltsp#425] [Q&A]
We have some remote workstations that traditionally have been local installs of Ubuntu, but since the offices are connected to our main network across fiber we would like to transition to LTSP thick clients. From time to time employees in the main office need to view the remote desktop, either for technical support or support with their job duties. Not everyone who helps them should have access to all desktops on our network, as epoptes provides, and they will be connecting from multiple workstations (some LTSP, some Macs). In the past we have run remote desktop sharing (vino) remotely and used remmina locally to view it. When I try to start vino through the gnome-control-center on Ubuntu 20.04 thick clients, it refuses to start. Running gnome-control-center from the command line shows the following error: "(gnome-control-center:26271): sharing-cc-panel-WARNING **: 14:50:14.358: Failed to enable service vino-server: GDBus.Error:org.freedesktop.DBus.Error.InvalidArgs: Sharing cannot be enabled on this network, status is '0'". Can anybody tell me how to get vino working, or recommend another program that allows VNC viewers to view specific LTSP desktops?

**Comment by alkisg** [comment]
Vino runs on Ubuntu GNOME 20.04 if using local installations, but not if if using LTSP?
I don't think that LTSP does anything that would interfere with vino.
Which LTSP version are you using?
Are you on Wayland or on Xorg?

Btw, how exactly do you want your vncviewers to authenticate to your vncservers? A VNC password for each PC? Or a VNC password for each target user? Or a set of clients for each supporter? Or an "accept incoming connection" dialog?

**Comment by mecrider** [comment]
I have LTSP v21.01-1 installed from the PPA, and am using Xorg. I've learned a little more, including a workaround. The GNOME Control Panel will let you set connection options, but won't start vino-server because it can't verify the network status from Network Manager (which shows I have an unmanaged wired connection). However, I can run /usr/lib/vino/vino-server from a terminal and it works just fine. So I add that command to Startup Applications and it works at each login. Currently I have a password set in vino-preferences (for that user) with no requirement to accept incoming connections.
